content/guides/accessibility-for-teams/content-design.md

```diff
@@ -136,124 +121,109 @@ Do your images have descriptive alt text?
 - Carmen’s page didn’t load all the way and didn’t get to download the images. 
 - Amanda is blind and uses a braille reader to understand the contents of images. 
 - John is looking for information with a search engine and needs help being directed to the right content (descriptive alt tags will improve search).
-
 {{< /ring >}}
 
 ### Steps to take
+
 - Include meaningful information describing each image in the alt text.
 - Use null (empty) alt text when text describing the image is already on the page (`alt=""`).
 - Don’t start the alt text with _Image of_ or _Photo of_ – the screen reader already announces that images are images.
 - If the image is decorative and you don’t want the screen reader to announce it at all, use  null (empty) alt text (`alt=""`).
 
-#### Resources
+### Resources
 
 {{< youtube XCa6U1BllCY >}}
```

Comment:
Set `resources` from `h4` to `h3` to be more consistent with section content.

Replaced `youtube` link with hugo `youtube` shortcode which embeds in page.